src/current/cockroachcloud/private-clusters.md

```diff
@@ -24,15 +24,15 @@ During [limited access](/docs/{{site.versions["stable"]}}/cockroachdb-feature-av
 
 ## Create a private cluster
 
-To create a private cluster, you must use [{{ site.data.products.db }} API](cloud-api.html) or [CockroachDB's Terraform provider](https://registry.terraform.io/providers/cockroachdb/cockroach/latest/docs/resources/cluster). When you [create](https://cockroachlabs.com/docs/api/cloud/v1/clusters) a private cluster using Terraform Provider, you set its `private_network_visibility` field to `true`. This in turn automatically sets uses the Cloud API to set the cluster's `network_visibility` field to `NETWORK_VISIBILITY_PRIVATE`.
+On GCP, new {{ site.data.products.dedicated }} advanced clusters are private by default.
+On AWS, newly {{ site.data.products.dedicated }} advanced clusters deployed on AWS are **not** private by default.
+
+To create a private cluster on AWS, you must use [{{ site.data.products.db }} API](cloud-api.html) or [CockroachDB's Terraform provider](https://registry.terraform.io/providers/cockroachdb/cockroach/latest/docs/resources/cluster). When you [create](https://cockroachlabs.com/docs/api/cloud/v1/clusters) a private cluster using Terraform Provider, you set its `private_network_visibility` field to `true`. This in turn automatically sets uses the Cloud API to set the cluster's `network_visibility` field to `NETWORK_VISIBILITY_PRIVATE`.
```

Comment:
As mentioned above, I think the UI is GA for this now. cc @jramos-crl

Reply:
Done

Reply:
confirming that yes, Dedicated advanced UI is GA.